diagnostic.md

```diff
@@ -14,7 +17,8 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+$('li').each('body');
+$('li').on('body')
```

Comment:
This will attempt to attach an event handler for a `body` event on all `<li>` elements.